build-system/tasks/prettify.js

```diff
@@ -48,43 +49,88 @@ const PrettierResult = {
 };
 
 /**
- * Checks files for formatting (and optionally fixes them) with Prettier.
- * Returns the cumulative result to the `gulp` process via  process.exitCode so
- * that all files can be checked / fixed.
+ * Extracts the list of prettifiable files in this PR from the commit log.
+ *
+ * @return {!Array<string>}
+ */
+async function getFilesChanged() {
```

Comment:
This is pretty similar to gulp lint's `lintableFilesChanged()`. Is there any overlap between the two that would be worth moving to `build-system/common/utils.js`?

Reply:
You're right. I did spend some time trying to unify the two, but they currently use different globbing mechanisms and a different filter. Okay to re-attempt this in a future PR that refactors this and other tooling functionality into `build-system/common/utils.js`?

Reply:
Wait, now that I read it again, I think it's reasonable to do it in this PR. Coming up!

Reply:
Done in the latest commit.